src/test/java/it/rate/webapp/services/InterestServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package it.rate.webapp.services;
+
+import it.rate.webapp.BaseTest;
+import it.rate.webapp.models.Criterion;
+import it.rate.webapp.models.Interest;
+import it.rate.webapp.repositories.CriterionRepository;
+import it.rate.webapp.repositories.InterestRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.same;
+import static org.mockito.Mockito.*;
+
+class InterestServiceTest extends BaseTest {
+
+  @MockBean InterestRepository interestRepository;
+  @MockBean CriterionRepository criterionRepository;
+
+  @Autowired InterestService interestService;
+
+  List<Criterion> criteria = new ArrayList<>();

```

Comment:
it is a bit dangerous keeping variables stored between the tests, you risk forgetting to reset them back to the previous state (see the comment below)